src/main/java/core/basesyntax/service/impl/ReportMakerServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportMakerService;
+
+public class ReportMakerServiceImpl implements ReportMakerService {
+    private static final String CSV_SEPARATOR = ",";
+    private static final String FRUITS_TITLE = "fruit";
+    private static final String QUANTITY_TITLE = "quantity";
+
+    @Override
+    public String makeReport() {
+        StringBuilder stringBuilder = new StringBuilder(rowMaker(FRUITS_TITLE, QUANTITY_TITLE));
+        Storage.fruits.entrySet()
+                .stream()
+                .forEach(entry -> stringBuilder.append(
+                        rowMaker(entry.getKey(),entry.getValue().toString())
+                ));
+
+        return stringBuilder.toString();
+    }
+
+    private String rowMaker(String firstColumn, String secondColumn) {
```

Comment:
use verbs for method naming
```suggestion
    private String makeRow(String firstColumn, String secondColumn) {
```